community/saas-companies-engagement-procedure.md

```diff
@@ -0,0 +1,118 @@
+## Procedure for Engaging With SaaS Companies
+
+### Summary
+
+This document outlines the procedure for engaging with SaaS companies, and
+how SaaS companies can represent themselves and their interests in the SPIFFE
+community.
+
+The goal is to provide a clear and consistent process for engaging with 
+SaaS companies that are included but not limited to those that use SPIFFE and 
+SPIRE, provide a proprietary SPIFFE-compatible service, or are interested in 
+contributing to the SPIFFE project.
+
+More specifically, this document includes guidelines for communication, 
+technical contact designation, and handling potential disruptive participants.
+
+### Technical Contacts
+
+#### Objective
+
+To ensure effective communication and support for SaaS consumers 
+using the SPIFFE specification by designating responsible technical contacts.
+
+#### Guidelines
+
+##### Designation of Contacts
+
+Each SaaS company that directs customers to the SPIFFE Slack channel must 
+designate two technical contacts.
+
+These contacts will serve as the primary point of contact for addressing queries 
+and facilitating communication between the customers and the Spiffy community.
+
+##### Documentation of Contacts
+
+A dedicated folder for each company will be created in the SPIFFE repository
+under the `./community/saas-technical-contacts` directory.
+
+The folder will contain:
+
+* Names and contact information of the designated technical contacts.
```

Comment:
It might be helpful here to define more specifically what "contact information" here needs to be. Can we provide a group email (e.g spiffe-community@goteleport.com) that can be directed to the appropriate contacts - or must this be specific contact details for these individuals?

Reply:
I think both individual contact email, or a distribution list would work — We’ll of course discuss this further within SSC. — and at any rate, nothing is set in stone, we can always amend this.

Thanks for all your insightful comments btw. Truly appreciate it.